File: src/main/kotlin/com/sourcegraph/cody/agent/protocol_generated/Attribution_SearchResult.kt
@file:Suppress("FunctionName", "ClassName", "unused", "EnumEntryName", "UnusedImport")
package com.sourcegraph.cody.agent.protocol_generated;

data class Attribution_SearchResult(
  val error: String? = null,
  val repoNames: List<String>,
  val limitHit: Boolean,
)

File: agent/bindings/csharp/AttributionSearchResult.cs
using System.Text.Json.Serialization;

namespace Cody.Core.Agent.Protocol
{
  public class AttributionSearchResult
  {
    [JsonProperty(PropertyName = "error")]
    public string Error { get; set; }
    [JsonProperty(PropertyName = "repoNames")]
    public string[] RepoNames { get; set; }
    [JsonProperty(PropertyName = "limitHit")]
    public bool LimitHit { get; set; }
  }
}
